theseus_gui/src/pages/project/Index.vue

```diff
@@ -344,16 +348,17 @@ const markInstalled = () => {
 async function install(version) {
   installing.value = true
   let queuedVersionData
+  metadatas.value = await getMetadatas(instance.value.path).catch(handleError)
 
   if (installed.value) {
     await remove_project(
       instance.value.path,
-      Object.entries(instance.value.projects)
+      Object.entries(metadatas.value)
         .map(([key, value]) => ({
           key,
           value,
         }))
-        .find((p) => p.value.metadata?.version?.project_id === data.value.id).key
+        .find((p) => p.value.version?.project_id === data.value.id).key
     )
```

Comment:
why was this needed? the instance projects are already available

Reply:
The instances are available but the version subfields are not, because when the instances are fetched the 'clear' field is passed as true to not load/reload all project data into memory, but those versions are needed for this. 

However I just double checked the backend and I think I missed how little data is contained in the projects that is not in the metadata, so it might be worth just flipping the true to a false so that all project data is loaded (or change it from only getting metadatas to only getting versions)